lib/stan_math_2.6.3/stan/math/prim/scal/prob/wiener_log.hpp

```diff
@@ -0,0 +1,229 @@
+// Copyright (c) 2013, Joachim Vandekerckhove.
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted
+// provided that the following conditions are met:
+//
+//   * Redistributions of source code must retain the above copyright notice,
+//   * this list of conditions and the following disclaimer.
+//   * Redistributions in binary form must reproduce the above copyright notice,
+//   * this list of conditions and the following disclaimer in the
+//   * documentation and/or other materials provided with the distribution.
+//   * Neither the name of the University of California, Irvine nor the names
+//   * of its contributors may be used to endorse or promote products derived
+//   * from this software without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+// LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+// CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
+// THE POSSIBILITY OF SUCH DAMAGE.
+
+#ifndef STAN_MATH_PRIM_MAT_PROB_WIENER_LOG_HPP
+#define STAN_MATH_PRIM_MAT_PROB_WIENER_LOG_HPP
+
+#include <stan/math/prim/scal/fun/constants.hpp>
+#include <stan/math/prim/scal/fun/square.hpp>
+#include <stan/math/prim/scal/fun/value_of.hpp>
+#include <stan/math/prim/scal/err/check_consistent_sizes.hpp>
+#include <stan/math/prim/scal/err/check_bounded.hpp>
+#include <stan/math/prim/scal/err/check_finite.hpp>
+#include <stan/math/prim/scal/err/check_not_nan.hpp>
+#include <stan/math/prim/scal/err/check_positive.hpp>
+#include <stan/math/prim/scal/meta/return_type.hpp>
+#include <stan/math/prim/scal/meta/include_summand.hpp>
+#include <boost/math/distributions.hpp>
+#include <cmath>
+#include <algorithm>  // for max
+
+namespace stan {
+
+  namespace math {
+
+     const double WIENER_ERR = 0.000001,
```

Comment:
I'd strongly prefer to see two const double declarations here.  What'd cpplint say?